awx/ui_next/src/components/AppContainer/BrandLogo.jsx

```diff
@@ -13,6 +13,8 @@ const BrandImg = styled.img`
   pointer-events: none;
 `;
 
-const BrandLogo = () => <BrandImg src="/static/media/logo-header.svg" />;
+const BrandLogo = ({ alt }) => (
```

Comment:
Do we need to provide a alt proper to all places that we are using `BrandLogo`? I just noticed the https://github.com/ansible/awx/blob/b8e05d6c6af3529284197e799d25dbfb6f1090bb/awx/ui_next/src/components/AppContainer/AppContainer.jsx#L161